openapi-docs/docs/DefaultApi.md

```diff
@@ -117,6 +118,55 @@ No authorization required
 - **Accept**: application/json
 
 
+## getNotificationProduct
```

Comment:
@vickywane I see a lot of references to `getNotificationProduct`. Is the intention to have a method hardwired to getting notification product information or is the goal to have a more generic `getProductInfo` method?

Reply:
I was yet to push the new updates from my local repository at the time of your review. I only called your attention to this [comment](https://github.com/ambianic/ambianic-subscriptions/pull/20#discussion_r654655850).